section2/reflection.md

```diff
@@ -2,14 +2,47 @@
+* A daily life conditional would be the process to decide what to wear in the morning before leaving. You would check to see if it will rain. If not you'd leave your rain coat. Will it be cold? if so maybe you bring a sweater. Will it be warm later in the afternoon? If so maybe you wear a layer under the sweater so you can remove it when it gets hot.
+* A conditional that is probably used on the website facebook for logging in could be something like
+
+```JavaScript
+var username = 'name-apellido'
+var password = 'abcde'
+
+if (username = 'name-apellido' && password = 'abcde') {
+console.log ('log in complete, please wait');
+} else if (username = 'name-apellido' && password !== 'abcde') {
+  console.log('password incorrect, please try again!');
+} else if (username !== 'name-apellido' && password = 'abcde') {
+console.log('username incorrect, please try again!');
+} else {
+  console.log('information incorrect. if you need help remembering click this link')
+}
+```
+
 5. How do you add multiple conditions to an `if` statement? In your own words, explain how to program reads them and determines what to do.
 
+* You add multiple conditions to an `if` statement by doing and `if/else if/else format`. In this format the computer will look at the first statement 'if....' and if it is determined to be true it will run just that section of code. If is it not true then it will check the next line. If that is evaluated and false then it moves onto the next line. This process continues until there is a condition that is met where the code is run. If none of the `else if` conditions are met then the computer will run the code for `else` at the end of the statement.
+
 6. What tools are available to you, as a developer, if you want to check that TWO conditions are met? Or, if you want to check that 1 of 2 conditions are met, before running a block of code?
 
-7. What questions do you still have about `if` statements and/or functions?
\ No newline at end of file
+*   The tools that are available if you want to check that two conditions are met are logical operators. More specifically the `&&` (and) logical operator. This will check if one variable and the other are true or not. If you wanted to check that that 1 of 2 conditions are met you would use the logical operator `||` (or). This will check whether one variable, or the other, or both are true or not.
+
+7. What questions do you still have about `if` statements and/or functions?
+
+I feel pretty confident with my understanding of conditionals and comparison operators up to this point. But, I still have questions about the use of logical operators. I understand the concept of them. How the `&&` operator basically checks whether multiple variables meet their conditions. And the `||` operator checks whether any of variables meet their conditions. But, I'm still not very clear with what they look like in real life examples in web applications.
```

Comment:
I work with transaction data at my company which gives cash back to our users, so an example could be if I am finalizing a transaction, I could want to know if the credit has been paid out and if a certain pending period has passed. An simplified example of that could look like this:
```js
if (transaction.isPaid && transaction.pendingPeriodDays > 7) {
    transaction.setFinalized
}
```